frontend/components/FrequentQuestions.vue

```diff
@@ -0,0 +1,70 @@
+<script setup lang="ts">
+import { mockData } from "~/shared/mockData";
+
+const props = defineProps({
+  questionsGroup: String,
+  title: String,
+});
```

Comment:
It's good practice to define the shape of Props and provide it as a type to `defineProps`:

```suggestion
interface Props {
    questionsGroup: string;
    title: string;
}

const props = defineProps<Props>();
```

Note that `string` instead of `String` is used to type strings in TypeScript. Both will work, but [there are subtle differences](https://stackoverflow.com/questions/14727044/what-is-the-difference-between-types-string-and-string). You might want to check the rest of your code for this and make the necessary replacements. :)